packages/test/test-end-to-end-tests/src/test/deRehydrateContainerTests.spec.ts

```diff
@@ -41,7 +41,7 @@ import { describeFullCompat } from "@fluid-private/test-version-utils";
 import {
 	getSnapshotTreeFromSerializedContainer,
 	// eslint-disable-next-line import/no-internal-modules
-} from "@fluidframework/container-loader/dist/utils.js";
+} from "@fluidframework/container-loader/lib/utils.mjs";
```

Comment:
This is a case where in the past we were testing CJS even though the test project is ESM. Now we are definitely testing the ESM output.